app/frontend/src/components/TextField.tsx

```diff
@@ -13,17 +15,42 @@ const useStyles = makeStyles((theme) => ({
       borderRadius: theme.shape.borderRadius,
     },
   },
+  minWidth: {
+    "& .MuiOutlinedInput-root": {
+      minWidth: 400,
```

Comment:
This may have broken the map location search bar in the edit profile page. I think it's better practise to set this to 100% and let the parent container that uses this control its width rather than hardcoding a min pixel value like this.

Reply:
That's why I exposed a `minWidth` prop that defaults to true.

The only problem with leaving `fullWidth` accessible like before is either losing or duplicating a lot the behaviour that sets it to 100% width on mobile only. 

And if we want to change the default desktop width, it would have to be changed in many places.

Reply:
I think that's ok since the text input has a different width requirement in different screens. For the edit profile we might want 400px but it looks like for the message view, it's meant to take up the full width of its container but that's not possible if we can't use `fullWidth`. How do you plan to fix that other than adding a custom override on the input there?